apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/localDateAssertions.kt

```diff
@@ -49,3 +49,26 @@ val <T : LocalDate> Expect<T>.month get(): Expect<Int> = ExpectImpl.localDate.mo
  */
 fun <T : LocalDate> Expect<T>.month(assertionCreator: Expect<Int>.() -> Unit): Expect<T> =
     ExpectImpl.localDate.month(this).addToInitial(assertionCreator)
+
+/**
+ * Creates an [Expect] for the property [LocalDate.getDayOfWeek] of the subject of the assertion,
+ * so that further fluent calls are assertions about it.
+ *
+ * @return The newly created [Expect].
+ *
+ * @since 0.10.0
+ */
+val <T : LocalDate> Expect<T>.dayOfWeek get(): Expect<Int> =
```

Comment:
As written in the description of the task, dies should return `Expect<DayOfWeek>`
I guess there are a few other changes due to this, I am not going to mark all of them.